src/System.Data.SqlClient/src/System/Data/SqlClient/LocalDBAPI.cs

```diff
@@ -48,45 +48,7 @@ internal static void ReleaseDLLHandles()
         private static IntPtr s_userInstanceDLLHandle = IntPtr.Zero;
 
         private static object s_dllLock = new object();
```

Comment:
Nit: readonly... I personally like to see readonly whenever possible, but in particular on lock objects, as if they're accidentally changed the results can be quite bad and hard to track down.